core/include/kernel/semihosting.h

```diff
@@ -0,0 +1,37 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2024 Andes Technology Corporation
+ */
+#ifndef __KERNEL_SEMIHOSTING_H
+#define __KERNEL_SEMIHOSTING_H
+
+#include <util.h>
+
```

Comment:
can remove this empty line

Reply:
Addressed in https://github.com/OP-TEE/optee_os/pull/6700/commits/e29f81b15fac69c9111d5b172c7b85a1f1444ed7